python/ml4ir/applications/ranking/tests/test_metrics_helper.py

```diff
@@ -0,0 +1,264 @@
+import unittest
+import json
+
+import pandas as pd
+from pandas import testing as pd_testing
+
+from ml4ir.applications.ranking.model.metrics.metrics_helper import *
+
+
+class ComputeSecondaryMetricsTest(unittest.TestCase):
```

Comment:
Some of these tests might seem redundant. But in some ways the point is that the existing metrics do not capture all scenarios and will be better captured with a metric that uses the actual values as opposed to binary assessments of the secondary label.